.circleci/config.yml

```diff
@@ -166,6 +166,30 @@ jobs:
       hostPath: graaljs
       hostName: graaljs
     <<: [*execution_steps]
+  "ESMeta: New or modified tests execution":
+    docker:
+      - image: mozilla/sbt:8u292_1.5.7
+    working_directory: ~/test262
+    environment:
+      hostType: esmeta
+      ESMETA_HOME: /app/esmeta
+    steps:
+      - checkout
+      - run: 
+          name: "Make directory"
+          command: mkdir -p "${ESMETA_HOME}" 
+      - run: 
```

Comment:
if possible we should run this on github actions, and NOT circle CI.

Reply:
That's totally possible. I'll work on an alternative PR running this on github actions.

Reply:
Great! Any reason not to update this one?

Reply:
Nope! I meant to work on a alternative version of this pr. I misworded.

I will force push the required commit.